esp/src/eclwatch/FileHistoryWidget.js

```diff
@@ -0,0 +1,119 @@
+/*##############################################################################
+#    HPCC SYSTEMS software Copyright (C) 2012 HPCC Systems®.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+############################################################################## */
+define([
+    "dojo/_base/declare",
+    "dojo/_base/lang",
+    "dojo/i18n",
+    "dojo/i18n!./nls/hpcc",
+    "dojo/_base/array",
+    "dojo/on",
+    "dojo/dom",
+    "dojo/dom-construct",
+    "dojo/dom-class",
+
+    "dijit/registry",
+    "dijit/ToolbarSeparator",
+    "dijit/form/Button",
+
+    "dgrid/selector",
+    "dgrid/extensions/ColumnResizer",
+
+    "hpcc/GridDetailsWidget",
+    "hpcc/WsDfu",
+    "hpcc/DelayLoadWidget",
+    "hpcc/ESPUtil"
+], function (declare, lang, i18n, nlsHPCC, arrayUtil, on, dom, domConstruct, domClass,
+                registry, ToolbarSeparator, Button,
+                selector, ColumnResizer,
+                GridDetailsWidget, WsDfu, DelayLoadWidget, ESPUtil) {
```

Comment:
Not used:
* on
* dom
* domConstruct
* domClass
* ToolbarSeparator
* Button
* DelayLoadWidget